src/resources/js/components/UserDeleteForm.svelte

```diff
@@ -0,0 +1,69 @@
+<script lang="ts">
+  import { Button, Helper, Checkbox } from 'flowbite-svelte';
+  import { createForm } from 'felte';
+  import { createEventDispatcher } from 'svelte';
+  import { ExclamationCircleOutline } from 'flowbite-svelte-icons';
+  import { validator } from '@felte/validator-yup';
+  import * as yup from 'yup';
+
+  import RootServerApi from '../lib/RootServerApi';
+  import type { User } from 'bmlt-root-server-client';
+  import { spinner } from '../stores/spinner';
+  import { translations } from '../stores/localization';
+
+  export let selectedUser: User | null;
+
+  const dispatch = createEventDispatcher();
+  let checkboxChecked: boolean = false;
+
+  const { form, errors } = createForm({
+    initialValues: { userId: selectedUser?.id, confirmCheckbox: false },
+    onSubmit: async () => {
+      spinner.show();
+      if (selectedUser) {
+        await RootServerApi.deleteUser(selectedUser.id);
+        dispatch('userDeleted', { userId: selectedUser.id });
+      }
+    },
+    onError: async (error) => {
+      console.log(error);
+      await RootServerApi.handleErrors(error as Error, {
+        handleValidationError: (error) => {
+          errors.set({
+            userId: (error?.errors?.userId ?? []).join(' ')
```

Comment:
This API will never throw validation errors, so you don't need to handle them. You can just do `await RootServerApi.handleErrors(error as Error)`.